publishedbranches/docs-mongocli.yaml

```diff
@@ -27,13 +29,14 @@ version:
     - '1.2'
     - '1.1'
     - '1.0'
-  stable: '1.8'
-  upcoming: '1.9'
+  stable: 'v1.9'
+  upcoming: 'v1.10'
 git:
   branches:
-    manual: 'v1.9'
+    manual: 'v1.10'
```

Comment:
Should I just remove this line if snooty doesn't use it?

Reply:
I think its fine to keep for now - we haven't rehauled these published branches files. So I think its ok to wait and just keep everything to be safe. I can always do clean up in future if we decide to formally get rid of a field

Reply:
Sounds good, thanks.